etc/init.d/contextBroker.ubuntu

```diff
@@ -0,0 +1,184 @@
+#! /bin/sh
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
```

Comment:
Looking to https://github.com/telefonicaid/fiware-orion/blob/develop/etc/init.d/contextBroker.centos, it seems that license use to be after the `### END INIT INFO` but not sure if this is actually needed or not (or if what is valid for CentOS is not valid for Ubunto or viceversa :)